src/components/ScrollBox/ScrollBox.tsx

```diff
@@ -0,0 +1,32 @@
+import { PropsWithChildren } from 'react';
+import { Box, BoxProps } from '@mantine/core';
+
+import styles from './ScrollBox.module.css';
+
+interface Props extends BoxProps, PropsWithChildren {
+  ref?: React.RefObject<HTMLDivElement | null>;
+  direction?: 'vertical' | 'horizontal' | 'both';
+}
+
+export function ScrollBox({ ref, direction = 'both', children, ...props }: Props) {
+  const vertical = direction === 'vertical' || direction === 'both';
+  const horizontal = direction === 'horizontal' || direction === 'both';
+  return (
+    <Box
+      ref={ref}
+      style={{
+        overflowX: horizontal ? 'auto' : 'hidden',
+        overflowY: vertical ? 'auto' : 'hidden'
+      }}
+      className={styles.scroller}
+      onWheel={(event) => {
+        if (horizontal) {
+          event.currentTarget.scrollLeft += event.deltaY;
+        }
```

Comment:
I don't think we have a panel that scrolls horizontally and vertically right now, but this behaviour will be quite strange if a panel has both. Cause then you would end up in the bottom right corner when scrolling down